modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Answer.es.js

```diff
@@ -313,9 +314,10 @@ export default withRouter(
 						<Comments
 							comments={comments}
 							commentsChange={_commentsChange}
-							companyMx={comments.companyMx}
+							companyName={context.companyName}
 							editable={editable}
 							entityId={answer.id}
+							hasCompanyMx={comments.companyMx}
```

Comment:
It should be hasCompanyMx, isn't? 

```suggestion
							hasCompanyMx={comments.hasCompanyMx}
```